omod/src/main/web/dashboardwidgets/programstatus/programstatus.controller.js

```diff
@@ -329,6 +331,7 @@ export default class ProgramStatusController {
             }).then((response) => {
                 if (this.config.disableReloadPage) {
                     this.fetchPatientProgram(this.patientProgram.uuid); // refresh display
+                    this.requestInProgress = false;
                 }
                 else {
                     this.reloadPage();  // closing a program affects other widgets, so we need to reload the entire page
```

Comment:
@mogoodrich actually I missed that but @icrc-toliveira pointed out that a subsequent reset of `this.requestInProgress` to `false` is made within `this.reloadPage()`.

So looks like it's ok to me 👍